lib/appoptics_apm/frameworks/sinatra.rb

```diff
@@ -27,6 +27,7 @@ def dispatch_with_appoptics
         ::AppOpticsAPM::API.log_exception('sinatra', e)
         raise e
       ensure
+        report_kvs[:Backtrace] = AppOpticsAPM::API.backtrace if AppOpticsAPM::Config[:sinatra][:collect_backtraces]
```

Comment:
Since it is a one liner, I think it is more readable to keep it. If I could abstract out the module name it may make sense. But at this point I don't think know if they are named consistently and I don't trust that they will be in the future.